geomesa-index-api/src/test/scala/org/locationtech/geomesa/index/geotools/GeoMesaDataStoreTest.scala

```diff
@@ -217,6 +220,57 @@ class GeoMesaDataStoreTest extends Specification {
         ds.stats.getMinMax[String](sft, "n", exact = false).map(_.max) must beSome("name0")
       }
     }
+    "update compatible schemas from typesafe config changes" in {
```

Comment:
Ok, could we add tests for:
1.  Showing that one cannot remove a field?
2.  Showing that an UnmodifiableUserDataKeys cannot be changed?
And maybe optional.... check the conditions on geometry field / column type changes?

Reply:
i think those are covered already by https://github.com/locationtech/geomesa/blob/main/geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreTest.scala#L349 (the schema tests are somewhat scattered around the codebase...)